cg/services/run_devices/pacbio/post_processing_service.py

```diff
@@ -49,11 +50,20 @@ def __init__(
         to_raise=PostProcessingError,
     )
     def post_process(self, run_name: str, dry_run: bool = False) -> None:
-        LOG.info(f"Starting PacBio post-processing for run: {run_name}")
+        LOG.info(f"Starting Pacbio post-processing for run: {run_name}")
         run_data: PacBioRunData = self.run_data_generator.get_run_data(
             run_name=run_name, sequencing_dir=self.sequencing_dir
         )
         self.run_validator.ensure_post_processing_can_start(run_data)
         self.store_service.store_post_processing_data(run_data=run_data, dry_run=dry_run)
         self.hk_service.store_files_in_housekeeper(run_data=run_data, dry_run=dry_run)
         self._touch_post_processing_complete(run_data)
+
+    def _get_all_run_names(self) -> list[str]:
+        """Get all run names in the Pacbio sequencing directory."""
+        run_names = []
+        for run_folder in Path(self.sequencing_dir).iterdir():
+            if run_folder.is_dir():
+                for cell_folder in run_folder.iterdir():
+                    run_names.append(f"{run_folder.name}/{cell_folder.name}")
+        return run_names
```

Comment:
this will crash if some random folder is added to the pacbio directory. I guess we should sanitise the directories somehow.

Reply:
Will end up with a reg ex check then I suppose...

Reply:
current folder in prod btw: 
![image](https://github.com/user-attachments/assets/66c74db4-b15a-43c4-838a-612fcf24c9e4)

Reply:
Should we not be able to rely on the data being correctly formatted in prod? To me I think we should remove the runs which break this logic.

Reply:
There are currently two folders in prod that do not seem to be tied to any specific run (`355`, `TransferSchemeTests`). We could tell the sync to hasta to ignore them and just assume that the folder will only contain runs.

That simplifies the logic but might make the command a bit brittle. What do you think?

Reply:
I think making sure we only process pac bio runs in the backend is more robust that relying on an assumption that can easily be violated by anyone who can run mkdir in prod.

Reply:
in this case the check would be to ignore folders that are not are not named properly like a run.
It would make sure the code should not reach a state where it breaks due to malformed data, which i think is more of a plus in an automated system. If the data within the expected directories is malformed then it should break of course but not due to that a wrong directory is read.

Reply:
I think I agree with Isak ☝️ Given that we proceed with the next run either way I think simplicity is prio 1